### ISSUE TITLE
Works on windows

### ISSUE THREAD

**Issue by ThaDafinser** [gophp7/gophp7-ext#11]
Check if it has a config.w32 file and works

http://gophp7.org/gophp7-ext/extensions-catalog.html

**Comment by auroraeosrose** [comment]
Might be prudent to label things as N/A on other platforms too - where you can probably do some clever porting of pcntl - there are parts of sysv that'll never work on windows - and the printer extension (for example) won't work on nix - so yeah it's a good idea to have this information but we also need the concept of "will never work" vs. "needs a port"

**Comment by ThaDafinser** [comment]
Pecl already provides windows builds if possible and there is a log, 
http://pecl.php.net/package/APC
http://windows.php.net/downloads/pecl/releases

Maybe we could take that as a source? Additionally there are logfiles, if the building is failing
//cc @weltling

**Comment by weltling** [comment]
@auroraeosrose @ThaDafinser @pierrejoye

That's clear that some stuff is platform dependent, like fork() or winapi. But in general IMHO @ThaDafinser is right, because ... Back then as we've started to do PECL builds for Windows, I've inspected almost every extension and made fixes directly or PRs where it made sense. It turned out, that from about 300 extensions on PECL, only 1/3 was somehow suitable for 5.5, furthermore maybe 1/2 of that 1/3 is really actively maintained. That's for Windows, but then the other factors like paltorm dependency, the extensions themselves or their underlying libraries come in addition. 

Still, for the overall goal, one can expect about 1/3 of the PECL extensions which are *almost* listed under http://windows.php.net/downloads/pecl/releases/ , so that were a good start. Some more could come in addition which is almost because they're platform specific (like uuid for example). Plus, it'd make sense to prioritize the list by their popularity/maintainability and whether it makes sense to port at all (for instance APC doesn't make sense for PHP7).

Also to be noted that the exts listed on PECL are probably not the only ones in the world. There are quite some at different places only like github, bitbucket, etc. Though to be noted that many of those are often not actively supported, often even don't have a release circle and there are no stats about how popular they are. Though some good exceptions do exist (uprofiler FE).

Cheers.
